runtime/compiler/control/J9Options.cpp

```diff
@@ -1137,6 +1139,14 @@ static bool JITServerParseCommonOptions(J9JavaVM *vm, TR::CompilationInfo *compI
       if (ret == OPTION_OK)
          compInfo->getPersistentInfo()->setSocketTimeout(timeoutMs);
       }
+   else if (xxJITServerRequireServerArgIndex >= 0)
```

Comment:
Doing the check on the `else` part of `if (xxJITServerTimeoutArgIndex >= 0)` means that if we specify a timeout we completely ignore the new option

Reply:
Yeah, that was the intention. Do you think the new option should take precedence? Or is it better to use the max of the two timeouts.

Reply:
If the user specified a timeout we should use that because the user knows best. But that doesn't mean that we should ignore the `requireJITServer` option.
The logic should be:
```
Look for +requireJITServer and -requireJITServer. The later one wins
If requireJITServer
   increase default timeout
if user specified timeout
   overwrite previous timeout value
```